src/IB/IBFESurfaceMethod.cpp

```diff
@@ -892,12 +892,15 @@ IBFESurfaceMethod::computeLagrangianForce(const double data_time)
 
         System* DP_system;
         const DofMap* DP_dof_map;
-        FEDataManager::SystemDofMapCache* DP_dof_map_cache;
+        // silence some warnings by giving DP_dof_map_cache a bogus, but
+        // valid, value if we don't use jump conditions
+        FEDataManager::SystemDofMapCache& DP_dof_map_cache =
+            d_use_jump_conditions ? *d_fe_data_managers[part]->getDofMapCache(PRESSURE_JUMP_SYSTEM_NAME) :
+                                    F_dof_map_cache;
```

Comment:
Can we please make it `nullptr` so that if we try to use it, it crashes instead of giving bizarre results?
```suggestion
                                    nullptr;
```

Reply:
I switched it to a reference instead of a pointer so I will have to go back and fix the type, but yes.

Reply:
Oh, you changed to a reference.

Can you keep it a pointer and explicitly initialize to `nullptr`?

Reply:
Yup